spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/autoconfig/OpenTelemetryAutoConfiguration.java

```diff
@@ -38,8 +40,16 @@ public class OpenTelemetryAutoConfiguration {
 
     @Bean
     @ConditionalOnMissingBean(SpanFactory.class)
-    public SpanFactory spanFactory() {
-        return OpenTelemetrySpanFactory.builder()
-                                       .build();
+    public SpanFactory spanFactory(TracingProperties properties) {
```

Comment:
I'd expect an additional test case in the `AxonAutoConfigurationWithOpenTelemetryTest` class.

Reply:
Added the test